src/bson/bson-config.h

```diff
@@ -174,6 +299,18 @@
 #endif
 
 
+/*
+ * Define to 1 if you have arc4random_buf available on your platform.
+ */
+#define BSON_HAVE_ARC4RANDOM_BUF 1
```

Comment:
We should only define this to 1 if we're sure arc4random_buf always exists on all platforms. Are we sure of that?

Reply:
I'm not sure of that and I'm having trouble finding info online. Will switch to 0.